package.json

```diff
@@ -48,11 +48,11 @@
     "socket.io": "^1.4.1",
     "socket.io-client": "^1.4.1",
     "steal-qunit": "0.0.4",
-    "steal-tools": "^0.13.0-pre.0",
+    "steal-tools": "^0.13.0",
     "testee": "^0.2.0"
   },
   "dependencies": {
-    "can": "^2.3.0-pre || ^2.3.0-beta || ^2.3.0",
+    "can": "git://github.com/canjs/canjs#master",
```

Comment:
i just want to test, if the current master branch with the fixes from mark stahl passes also can-ssr.
so thr problem was indeed jquery 2.2.